src/auth/auth.service.ts

```diff
@@ -1,7 +1,7 @@
 import { Injectable } from '@nestjs/common';
 import { UsersService } from '../users/users.service';
 import { JwtService } from '@nestjs/jwt';
-import * as bcrypt from 'bcrypt';
+import * as argon2 from 'argon2';
```

Comment:
**praise**: Switching from bcrypt to argon2 for password hashing is a great choice for enhanced security.